x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/authorization/alerts/essentials_tier/admin.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import TestAgent from 'supertest/lib/agent';
+import {
+  DETECTION_ENGINE_ALERT_ASSIGNEES_URL,
+  DETECTION_ENGINE_ALERT_TAGS_URL,
+  DETECTION_ENGINE_QUERY_SIGNALS_URL,
+  DETECTION_ENGINE_SIGNALS_STATUS_URL,
+} from '@kbn/security-solution-plugin/common/constants';
+import {
+  createAlertsIndex,
+  createRule,
+  deleteAllAlerts,
+  deleteAllRules,
+  getAlertsByIds,
+  getRuleForAlertTesting,
+  waitForAlertsToBePresent,
+  waitForRuleSuccess,
+} from '../../../../../../../common/utils/security_solution';
+import { getAlertStatus, setAlertAssignees, setAlertStatus, setAlertTags } from '../../../../utils';
+import { FtrProviderContext } from '../../../../../../ftr_provider_context';
+import { EsArchivePathBuilder } from '../../../../../../es_archive_path_builder';
+
+const query = {
+  ...getAlertStatus(),
+  query: {
+    bool: {
+      should: [{ match_all: {} }],
+    },
+  },
+};
+
+export default ({ getService }: FtrProviderContext): void => {
+  const log = getService('log');
+  const utils = getService('securitySolutionUtils');
+  const esArchiver = getService('esArchiver');
+  const es = getService('es');
+  const config = getService('config');
+  const isServerless = config.get('serverless');
+  const dataPathBuilder = new EsArchivePathBuilder(isServerless);
+  const path = dataPathBuilder.getPath('auditbeat/hosts');
+
+  let admin: TestAgent;
+
+  describe('@serverless @serverlessQA admin alerts API behaviors', () => {
+    before(async () => {
+      await esArchiver.load(path);
+      admin = await utils.createSuperTest('admin');
+    });
+
+    beforeEach(async () => {
+      await deleteAllRules(admin, log);
+      await createAlertsIndex(admin, log);
+    });
+
+    after(async () => {
+      await esArchiver.unload(path);
+    });
+
+    afterEach(async () => {
+      await deleteAllAlerts(admin, log, es);
+    });
+
+    describe('find alerts', () => {
+      it('should return 200 for admin', async () => {
+        await admin
+          .post(DETECTION_ENGINE_QUERY_SIGNALS_URL)
+          .set('kbn-xsrf', 'true')
+          .send(query)
+          .expect(200);
+      });
+    });
+
+    describe('set alert tags', () => {
+      it('should return 200 for admin', async () => {
+        const rule = {
+          ...getRuleForAlertTesting(['auditbeat-*']),
+          query: 'process.executable: "/usr/bin/sudo"',
+        };
+        const { id } = await createRule(admin, log, rule);
+        await waitForRuleSuccess({ supertest: admin, log, id });
+        await waitForAlertsToBePresent(admin, log, 10, [id]);
+        const alerts = await getAlertsByIds(admin, log, [id]);
+        const alertIds = alerts.hits.hits.map((alert) => alert._id!);
```

Comment:
Nit: if the details of creating alerts aren't particularly relevant to the tests (as they seem not to be here), I'd consider moving this into some kind of helper `generateRuleAndAlerts` that we can call here.